template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Controllers/IdentityServiceController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.Service;
+using Microsoft.AspNetCore.Identity.Service.IntegratedWebClient;
+using Microsoft.AspNetCore.Identity.Service.Mvc;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Company.WebApplication1.Identity.Models;
+
+namespace Company.WebApplication1.Identity.Controllers
+{
+    [Area(IdentityServiceConstants.Tenant)]
+    [Route("tfp/[area]/" + IdentityServiceConstants.DefaultPolicy + "/oauth2/v" + IdentityServiceConstants.Version + "/[action]")]
+    public class IdentityServiceController : Controller
```

Comment:
Tracking this discussion with https://github.com/aspnet/IdentityService/issues/91